servers/rendering_server.cpp

```diff
@@ -2183,6 +2183,7 @@ void RenderingServer::_bind_methods() {
 	/* VIEWPORT */
 
 	ClassDB::bind_method(D_METHOD("viewport_create"), &RenderingServer::viewport_create);
+	ClassDB::bind_method(D_METHOD("viewport_set_use_xr", "viewport", "use_xr"), &RenderingServer::viewport_set_use_xr);
```

Comment:
https://github.com/godotengine/godot/pull/70970#discussion_r1091726299
This fixed the issue in Linux CI, the setter is exposed only at `RenderingServer` level. I think we can remove the break_compatibility tag.